changelog.md

```diff
@@ -42,6 +42,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 * Models page:
   * Rework on tag display, use tag category for all diagrams in table.
   * Rework on tag display, use tag language for plugin selection on create diagram from template.
+  * Update ModelizerSettingsMenu to display user info.
```

Comment:
Update naviguation bar to display user info

Reply:
done